scripts/setup_debian_rootfs.sh

```diff
@@ -28,6 +30,10 @@ while test -n "$1"; do
     LOOP_DEVICE_ID=$2
     shift
     ;;
+  -p | --password)
+      DISK_PASSWORD=$2
+      shift 2
+      ;;
```

Comment:
shouldn't it be --mapper-name and MAPPER_NAME?

Reply:
Yes indeed, so much copy-paste hahaha